src/mumble/MainWindow.cpp

```diff
@@ -505,12 +505,13 @@ bool MainWindow::nativeEvent(const QByteArray &, void *message, long *) {
 
 void MainWindow::closeEvent(QCloseEvent *e) {
 	ServerHandlerPtr sh = Global::get().sh;
+	QuitBehavior quitBehavior = Global::get().s.quitBehavior;
 
+	if (!bSuppressAskOnQuit && ((quitBehavior == QuitBehavior::ALWAYS_ASK)
+			|| (sh && sh->isRunning() && quitBehavior == QuitBehavior::ASK_WHEN_CONNECTED))) {
```

Comment:
```suggestion
	const bool alwaysAsk = quitBehavior == QuitBehavior::ALWAYS_ASK;
	const bool askDueToConnected = sh && sh->isRunning() && quitBehavior == QuitBehavior::ASK_WHEN_CONNECTED;

	if (!bSuppressAskOnQuit && (alwaysAsk || askDueToConnected)) {
```
Imo that makes things a bit more readable.